release-1.8/release_notes_draft.md

```diff
@@ -44,3 +44,37 @@ please check out the [release notes guidance][] issue.
 ## **Deprecations**
 
 ## **Notable Features**
+
+### **Node Components**
+#### kubelet
```

Comment:
I get the usage here, but workload placement sounds to me like something in the scheduler domain. How about instead of "workload placement control" we go for "differentiated compute quality-of-service"?